chart/byconity/templates/daemonmanager-configmap.yaml

```diff
@@ -41,6 +41,8 @@ data:
     {{- $rm_service := dict "service" (printf "%s-%s" .Release.Name "resource-manager") "headless_service" (printf "%s-%s-headless" .Release.Name "resource-manager") "psm" (printf "%s-%s" .Release.Name "resource-manager") }}
     {{- $service_discovery := dict "service_discovery" (dict "server" $server_service "tso" $tso_service "daemon_manager" $dm_service "resource_manager" $rm_service) }}
     {{- $catalog := dict "catalog" (dict "name_space" (include "byconity.fullname" .)) }}
-    {{- $hdfs_addr := dict "hdfs_addr" .Values.byconity.hdfs_address }}
+    {{- $hdfs_addr_dict := dict "hdfs_addr" .Values.byconity.hdfs_address }}
+    {{- $hdfs_ha_nameservice_dict := dict "hdfs_ha_nameservice" .Values.byconity.hdfs_ha_nameservice }}
```

Comment:
I think the system won't read hdfs config from here

Reply:
Are you worried that it won’t properly choose between hdfs_address and hdfs_ha_nameservice here? The code I’m running is working fine, you can try it. Of course, I also need to check if all the components are covered.

Reply:
@dmthuc  I’m not sure what you mean. Why don’t you add me on WeChat: `cheng1483,` it would be easier to communicate this way.

Reply:
Hi @FourSpaces , I've added to WeChat. Let's discuss